virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -9,6 +9,13 @@ Before you install {VirtProductName}, ensure that your {product-title} cluster m
 * Your cluster must be installed on
 xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure with Red Hat Enterprise Linux CoreOS workers.
 
+* Additionally, your cluster must use xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[deploy machine health checks] to maintain high availability (HA).
```

Comment:
I would add: high availability of *virtual machines*. Because "high availability" can mean many things.